docs/docs/external-volumes.md

```diff
@@ -200,12 +200,12 @@ The default implicit volume size is 16 GB. If you are using the Mesos containeri
  [1]: /administration/installing/custom/cli/
  [2]: /administration/installing/custom/advanced/
  [3]: http://rexray.readthedocs.io/en/v0.3.3-rc1/user-guide/storage-providers/
```

Comment:
This should be https://rexray.readthedocs.io/en/v0.3.3/user-guide/storage-providers/.